Makefile

```diff
@@ -252,11 +252,8 @@ tmabci = third_party/proto/tendermint/abci/types/types.proto
 proto-update-tendermint:
```

Comment:
Can we have something more like this instead? 

https://github.com/cosmos/gaia/pull/285/files#diff-b67911656ef5d18c4ae36cb6741b7965R243